app/src/main/java/io/neurolab/activities/MeditationActivity.java

```diff
@@ -44,6 +45,14 @@ protected void onCreate(Bundle savedInstanceState) {
         initializePlaybackController();
     }
 
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
```

Comment:
Out of curiosity, isn't it just enough to set a landscape resource file without having to manually call a setview? :thinking:

Reply:
Yes @dipu989 We can simply make the `land` layout.

Reply:
@CloudyPadmal @jddeep I did the same before! But the ScrollView was only working when I would enter Meditation Activity in Landscape mode.
If I enter in the meditation activity in Portrait mode and then switch to landscape mode, it was not working there!
So I have to use ```onConfigurationChanged()``` here.

Reply:
@dipu989 Refer to this: https://github.com/fossasia/neurolab-android/pull/487

Reply:
@jddeep  Yes, I have looked into #487! XML is styled the same way but behaviour is not as expected.
The above quoted problem persists.

Reply:
@dipu989 let me have a look, give me time till tomorrow

Reply:
@adityastic Will you please review this PR?